catroid/src/main/java/org/catrobat/catroid/ui/fragment/ProjectListFragment.java

```diff
@@ -399,4 +400,10 @@ public void onLoadProjectSuccess(boolean startProjectActivity) {
 	public void onLoadProjectFailure() {
 		getActivity().findViewById(R.id.fragment_container).setVisibility(View.VISIBLE);
 	}
+
+	@Override
+	public void onDetach() {
+		super.onDetach();
+		loadProjectTask.cancel(true);
```

Comment:
Yes, it throws NullPointerException. Applied a null checker for loadProjectTask.